packages/ui/useScroll/useScroll.js

```diff
@@ -287,6 +288,29 @@ const useScrollBase = (props) => {
 		}
 	}, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+	useLayoutEffect(() => {
+		const containerRef = scrollContainerRef.current;
+		if (!containerRef) {
+			return;
+		}
+		if (typeof ResizeObserver === 'function') {
+			let resizeObserver = new ResizeObserver(() => {
+				if (scrollContentHandle.current && scrollContentHandle.current.syncClientSize) {
+					scrollContentHandle.current.syncClientSize();
+				}
+			});
+
+			resizeObserver.observe(containerRef);
```

Comment:
No need to do resizeObserver.unobserve()?

Reply:
I did disconnect().

Reply:
Ah, I saw resizeObserver.disconnect()